src/dataflow/internal/process/functions/source.ts

```diff
@@ -6,7 +7,7 @@ import {overwriteEnvironments} from '../../../environments'
 import type {DataflowInformation} from '../../info'
 
 export function processSourceCall<OtherInfo>(functionCall: RFunctionCall<OtherInfo & ParentInformation>, data: DataflowProcessorInformation<OtherInfo & ParentInformation>, information: DataflowInformation): DataflowInformation {
-	const sourceFile = functionCall.arguments[0]
+	const sourceFile = functionCall.arguments[0] as RArgument<ParentInformation>
```

Comment:
actually i meant it the other way around as the 0th index can, theoretically, be undefined for if we encounter `source()`

Reply:
well you should've said so

Reply:
but i'm checking for that in the `if` right below: `sourceFile?.value?.type == RType.String`